lib/src/widget/bottom_sheet/bottom_sheet_base.dart

```diff
@@ -29,6 +29,7 @@ class _BottomSheetBaseState extends State<BottomSheetBase>
       padding: widget.padding ??
           EdgeInsets.symmetric(
             horizontal: linden.dimen.unit3,
+            vertical: linden.dimen.unit3,
           ),
```

Comment:
```suggestion
          EdgeInsets.all(linden.dimen.unit3),
```

Reply:
Sorry, had to change it again. It's now different :)